### ISSUE TITLE
Regarding of the model details of BERT  stated in the PAWS paper

### ISSUE THREAD

**Issue by anbo724** [google-research-datasets/paws#7]
As mentioned in the paper, the bert model is fine-tuned with just one additional output layer. Can you give more detail information about the layer? Or show the code of the model?
Thanks.

**Comment by yuanzh** [comment]
The layer is just a softmax layer with output dimension = number of classes. E.g. for paraphrase identification the output dimension = 2 (same meaning or not same meaning). The input of the layer is the CLS encoding from BERT. Hope this helps.

**Comment by anbo724** [comment]
Thanks for your reply, for each sentence pair, the model learns two vectors for each sentence, or concatenate the two sentences like [CLS] sentence1 [SEP] sentence2 [SEP] ?
Thanks.

**Comment by yuanzh** [comment]
Concatenate the two sentences like [CLS] sentence1 [SEP] sentence2 [SEP]

**Comment by anbo724** [comment]
Thanks！
